src/executors/default.yml

```diff
@@ -13,6 +13,10 @@ parameters:
     type: enum
     enum: ["small", "medium", "medium+", "large", "xlarge", "2xlarge", "2xlarge+"]
     default: "medium"
+  circleci_ip_ranges:
```

Comment:
What is this parameter for?

Reply:
good catch, these parameters are not needed in the executors